lib/mapObjects/CGHeroInstance.h

```diff
@@ -108,7 +108,8 @@ class DLL_LINKAGE CGHeroInstance : public CArmedInstance, public IBoatGenerator,
 		}
 	};
 
-	std::vector<HeroSpecial*> specialty;
+	std::vector<HeroSpecial*> specialtyDeprecated;
+	CBonusSystemNode specialty;
```

Comment:
Is this field used?

Reply:
no, removed (both specialtyDeprecated and specialty).